content/strategy-goals/strategy/own/index.md

```diff
@@ -0,0 +1,38 @@
+# Own Strategy
+
+## Own elevator pitch
+
+Sourcegraph Own is a revolutionary way to manage code ownership for all your code across repositories and code hosts. It allows you to react to incidents and security issues faster and increases developer productivity by making it easy to find who owns the code quickly. Unlike CODEOWNERS, Soucegraph Own is always up to date, provides 100% ownership coverage, and connects to enterprise systems (LDAP, incident management, etc). Own tells you about who’s accountable for code and who knows about code.
+
+## Quick links
+
+- [Engineering strategy](../../../departments/engineering/index.md#product-vision-and-strategy)
+- [Own](../../../departments/engineering/teams/own/index.md)
+- [Product Marketing Brief](https://docs.google.com/document/d/1597R1KNoyrU79twn0xoBBMxexnRRGe2I1VQA4mxtR-I/edit#heading=h.a9uhc0ymvnxt)
+
+## Vision
+
+In 3 years, nobody will have to worry about CODEOWNERS being out of date. Devs will be able to quickly understand who owns what code and service, and how all the pieces fit together, and collaboration will be so much easier. The People Graph and the Source Graph will be merged.
+
+## Mission
+
+Allow developers to quickly answer questions about relationships between code, and the persons who build or interact with it. Make all the workflows that require navigating the org chart to improve or fix code easy. Reduce noise, automate boring tasks, kill useless emails.
+
+## Strategy
+
+Own is an emerging product, that we plan to release as an MVP at Starship in March 2023. Our product strategy is relatively bare-bone at this stage:
+
+- First, we'll lay some foundatations. Sourcegraph Own will ingest existing ownership data (eg. from CODEOWNERS) and provide powerful ways to use it that neither CODEOWNERS nor in-house systems provide, building on the power of Sourcegraph.
+  - What code does alice own that has the AuthParser class? Search for `AuthParser file:has.owner(@alice)`.
+  - We need to remove AuthParser from the codebase. Who owns code that uses it? Search for `AuthParser select:owner`, get a list of all the owners, quickly contact them.
+  - Search for files without owner
+- Second, we'll make ownership 10x easier and 10x more valuable.
```

Comment:
this and beyond is not march 2023, correct?

Reply:
This is strategy, so is 1 year+ ahead (less would be good, too!)